### ISSUE TITLE
Question: [How to format specific text in TextBox]

### ISSUE THREAD

**Issue by jeganraj-m** [microsoft/microsoft-ui-xaml#6960]
In Windows 11 OS, search textbox displays as below
![image](https://user-images.githubusercontent.com/74640452/162579788-21351d18-97c2-41ab-bf8b-16acefdc4c0d.png)

Here the auto append text has a different foreground. How to do this in WinUI TextBox?

**Comment by DmitryBorodiy** [comment]
Are you talking about the TextBox. PlaceholderText property?

**Comment by jeganraj-m** [comment]
No, PlaceholderText displays when there is no text in TextBox. What i'm asking is while typing in TextBox, I need to show the suggestion as like in Win 11 OS.

Currently I have behavior as like below

![image](https://user-images.githubusercontent.com/74640452/162584314-bc93fef1-e44c-4a2c-87be-336fb56a5e18.png)

I need to change it like below or as in Win 11 UI
![image](https://user-images.githubusercontent.com/74640452/162584394-239d9bcf-3418-4083-a657-3d6eb8df877e.png)

**Comment by chingucoding** [comment]
I dont think there is built in behavior for this. You could achieve this by retemplating (not sure how easy that would be) or put another textblock beneath the AutoSuggestBox/TextBox and achieve the effect that way.

**Comment by ojhad** [comment]
Textbox does not have this behavior by default. An option to achieve this is to retemplate RichEditBox.

**Comment by AndrewKeepCoding** [comment]
Hi @jeganraj-m! I took the liberty and create an [AutoCompleteTextBox](https://github.com/AndrewKeepCoding/AK.Toolkit)  from a custom control. It show the suggestion inside the TextBox just like you required by overriding the placeholder feature. You can get as NuGet package or just take the code in the repo.
